test/task/CleanStoppedSessionTask.test.ts

```diff
@@ -66,6 +67,7 @@ describe('CleanStoppedSessionTask', () => {
 
   beforeEach(async () => {
     domMockBuilder = new DOMMockBuilder(behavior);
+    browserBehavior = new DefaultBrowserBehavior();
```

Comment:
Should there be a `afterEach` to set the `browserBehavior` to `undefined`. Not sure if this will keep stale objects when running unit tests.

Reply:
Same for other tests included in this change.